shared/src/main/scala/io/github/hamsters/EmptyOptionValues.scala

```diff
@@ -0,0 +1,22 @@
+package io.github.hamsters
+
+object EmptyOptionValues {
+  implicit class StringOptionValues(optStr: Option[String]){
```

Comment:
StringOptionValues shoud not be plural as there is a single default value for empty Option[String].
Also it should reflect the fact that it's a value for an empty option.
(same comment for other implicit classes)

Reply:
What do you think of this name: EmptyStringOptionValue

Reply:
👍

Reply:
This is a brilliant approach. I'll refactor accordingly :)